google-cloud-video_intelligence/google-cloud-video_intelligence.gemspec

```diff
@@ -20,7 +20,8 @@ Gem::Specification.new do |gem|
 
   gem.required_ruby_version = ">= 2.0.0"
 
-  gem.add_dependency "google-gax", "~> 0.8.0"
+  gem.add_dependency "google-gax", "~> 0.8.6"
+  gem.add_dependency "googleapis-common-protos", "~> 1.3.1"
```

Comment:
The `google-gax` gem already [declares](https://rubygems.org/gems/google-gax/versions/0.8.6) a dependency on `googleapis-common-protos` with `~> 1.3.5`, so I don't think this line is necessary.

Reply:
This is autogenerated -- it's configured at as an API-level dependency because the API's protos depend on common protos, so this GAPIC depends directly on types defined in the common protos gem. It should be harmless, right?

Reply:
Yes, this is unnecessary and harmless.

Reply:
Acknowledged. I would rather leave this in, then. It probably does not make sense for the code generator to be aware of GAX's dependencies.